common/lib/api-client/middleware/request-include.js

```diff
@@ -0,0 +1,26 @@
+const { get } = require('lodash')
+
+module.exports = {
+  name: 'req-include',
+  req: function req(payload = {}) {
+    if (!payload.req) {
+      return payload
+    }
+
+    const { req, jsonApi = {} } = payload
+    const defaultInclude = get(
```

Comment:
I would have only bothered to look this up if there is no `include` param ;)

Mind you, I'd probably have unnecessarily cached it…